library/src/G4CMPPhononBoundaryProcess.cc

```diff
@@ -311,13 +325,15 @@ GetReflectedVector(const G4ThreeVector& waveVector,
       << ", newNorm = " << newNorm
       << ", reflectedKDir = " << reflectedKDir
       << ", kPerpV (kPerpMag * newNorm) = " << kPerpV
-      << ", kTan (reflectedKDir - kPerpV) = " << kTan << G4endl;
+      << ", kTan (reflectedKDir - kPerpV) = " << kTan
+      << ", surfaceStepSize = " << stepSize
```

Comment:
You could use `G4BestUnit(stepSize)` here (from `#include "G4UnitsTable.hh"`), or you could explicitly choose mm or um or something.